src/pages/BrowseSchemes.svelte

```diff
@@ -179,6 +180,7 @@
           </CollapsibleCard>
           <CollapsibleCard label="Census">
             <CensusOutputAreaLayerControl />
+            <ImdLayerControl />
```

Comment:
It's possible to one of three OA-level census layers alongside the IMD layer. This is awkward, but I don't know how to solve it (UX-wise) yet. I think we only ever want one boundary/census/"covers everything" layer on at a time, and not sure how to communicate that as radio buttons across categories

Reply:
I agree. I don't know how to communicate it, and it's worth checking with users first, but making these and boundaries all mutually exclusive sounds like a good future task